src/utils/daos_hdlr.c

```diff
@@ -54,11 +54,92 @@ parse_acl_file(const char *path, struct daos_acl **acl);
  * int pool_stat_hdlr(struct cmd_args_s *ap);
  */
 
+static int
+pool_decode_props(daos_prop_t *props)
+{
+
+	struct daos_prop_entry          *entry;
+	int 				rc = 0;
+
+	/* unset properties should get default value */
+
+	entry = daos_prop_entry_get(props, DAOS_PROP_PO_LABEL);
+	if (entry == NULL || entry->dpe_str == NULL) {
+		fprintf(stderr, "label property not found\n");
+		D_GOTO(out_err, rc = -DER_INVAL);
```

Comment:
From the user's perspective, it might be better to print something useful like "not set" or "(null)", instead of exiting with an error if any of these properties comes back NULL for some reason.

Reply:
@kjacque my understanding is that all properties should be available at least with default values, so IMO this is a real error case. BTW, this was the original behaviour already before my change, so ...

Reply:
@kjacque I am not sure I get this, do you mean that this kind of error msg could puzzle users ? But then should we redirect them to the log ? And try to continue ? And finally produce truncated output with an exit code ? And yes I did not change/add this behaviour.

Reply:
Yeah, my concern is that we stop processing the props and exit if we encounter these errors. I think it would be better to try to display all of them. I think it's more helpful for troubleshooting what's wrong, and if a prop gets set to NULL somehow, it shouldn't impact the user's ability to inspect the other props.

Reply:
> And yes I did not change/add this behaviour.

eh eh ... I did not change/add this behaviour within this patch, but I now remember that I am his originator !! :-)